mira/dkg/construct.py

```diff
@@ -465,19 +460,11 @@ def extract_ontology_subtree(curie: str, add_subtree: bool = False):
         type = "class"
         version = get_version(resource_prefix)
         cached_relabeled_obo_graph_path = prefix_directory_join(resource_prefix,
-                                                   name="relabeled_obo_graph.pkl",
-                                                                 version=version)
-        if not cached_relabeled_obo_graph_path.exists():
-            _, obo_path = _ensure_ontology_path(resource_prefix, force=False,
-                                                version=version)
-            obo_graph = read_obo(obo_path)
-            relabeled_graph = networkx.relabel_nodes(obo_graph,
-                                               lambda node_index: node_index.lower())
-            with open(cached_relabeled_obo_graph_path,'wb') as relabeled_graph_file:
-                pickle.dump(relabeled_graph, relabeled_graph_file)
-        else:
-            with open(cached_relabeled_obo_graph_path,'rb') as relabeled_graph_file:
-                relabeled_graph = pickle.load(relabeled_graph_file)
+                                                            name="relabeled_obo_graph.pkl",
+                                                            version=version)
+
+        with open(cached_relabeled_obo_graph_path,'rb') as relabeled_graph_file:
```

Comment:
I think at this point we should still have a fallback where if the file doesn't exist, we call `download_convert_ncbitaxon_obo_to_graph`. Of course in the context of Docker, the file will already be there but it may not be in all environments.